controllers/imagecollector/imagecollector_controller.go

```diff
@@ -353,3 +396,31 @@ func (r *Reconciler) handleCompletedImageJob(ctx context.Context, req ctrl.Reque
 
 	return ctrl.Result{RequeueAfter: *repeatPeriod}, err
 }
+
+func recordMetrics(ctx context.Context, jobDuration float64, podsCompleted int64, podsFailed int64) {
+	p := global.MeterProvider()
+
+	duration, err := p.Meter("eraser").SyncFloat64().Histogram("ImageJobCollectorDuration", instrument.WithDescription("duration of collector imagejob"), instrument.WithUnit(unit.Milliseconds))
+	if err != nil {
+		panic(err)
```

Comment:
i don't think we need to panic if there's an error in logs, let's return an error